inst/stan/functions/expected_obs_from.stan

```diff
@@ -143,7 +142,7 @@ vector expected_obs_from_snaps(int start, int end, array[] vector imp_obs,
       esnap += l;
       log_exp_obs[ssnap:esnap] = expected_obs_from_index(
         i, imp_obs, rdlurd, srdlh, refp_lh, dpmfs, ref_p, rep_h, ref_as_p, g, t,
-        l, refnp_lh, ref_np, p
+        l, refnp_lh, ref_np, p, rep_agg_probs, rep_agg_indicators[g, t][1:l, 1:l]
```

Comment:
added the indexing to expected_obs_from_snaps() here, assuming I just needed to pull it "up" a level from expected_obs_from_index? difficult for me to wrap my head around what to be pulling where

Reply:
This looks right to me